Engine/Engine/Core/Renderer.cpp

```diff
@@ -92,18 +92,14 @@ void Renderer::CreateBasicProgram()
 		delete[] infoLog;
 	}
 
-	glm::vec3 eye = glm::vec3(0, 0, 0);
-	glm::vec3 center = eye * glm::vec3(0, 0, 1);
-	glm::vec3 up = glm::vec3(0, 1, 0);
-	lookMat = glm::lookAt(eye, center, up);
-
-	persMat = glm::perspective(3.14159f * 0.4f / 1.0f, 1920.f / 1080.f, 0.01f, 1000.f);
-
-	//glUniformMatrix4fv(4, 1, GL_FALSE, &lookMat[0][0]);
-	//glUniformMatrix4fv(5, 1, GL_FALSE, &persMat[0][0]);
-
-	testMesh = new Mesh(am->GetMesh("box"));
-
+	obj1 = new Mesh(am->GetMesh("box"), glm::vec3(0, 0, 5));
+	obj2 = new Mesh(am->GetMesh("sphere"), glm::vec3(5, 0, 0));
```

Comment:
I don't have access to these models. (sphere, sword, teapot)  Can't test without them.